Testing packaged version failed https://github.com/MDAnalysis/GridDataFormats/actions/runs/21233099346/job/61095392408

```
FAILED tests/test_mrc.py::test_mrcfile_volume_check - AssertionError: Regex pattern did not match.
  Expected regex: 'is not a volumetric density'
  Actual message: 'Wrong/missing data to set up Grid. Use Grid() or Grid(grid=<array>, edges=<list>) or Grid(grid=<array>, origin=(x0, y0, z0), delta=(dx, dy, dz)):\ngrid=/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/gridData/tests/datafiles/ispg_0.mrc edges=None origin=None delta=None'

FAILED tests/test_mrc.py::test_mrcfile_volume_force - ValueError: Wrong/missing data to set up Grid. Use Grid() or Grid(grid=<array>, edges=<list>) or Grid(grid=<array>, origin=(x0, y0, z0), delta=(dx, dy, dz)):
grid=/opt/hostedtoolcache/Python/3.14.2/x64/lib/python3.14/site-packages/gridData/tests/datafiles/ispg_0.mrc edges=None origin=None delta=None
```

--- a/.github/workflows/deploy.yaml
+++ b/.github/workflows/deploy.yaml
@@ -10,7 +10,7 @@ on:
 
 concurrency:
   group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
-  cancel-in-progress: true
+  cancel-in-progress: false
 
 defaults:
   run:
@@ -28,7 +28,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.x"
+          python-version: "3.14"
 
       - name: Install build dependencies
         run: |
@@ -50,31 +50,6 @@ jobs:
           path: dist/
           retention-days: 1
 
-  test-install:
-    name: Test package installation
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: "3.x"
-
-      - name: Download dist files
-        uses: actions/download-artifact@v4
-        with:
-          name: dist-files
-          path: dist/
-
-      - name: Install package
-        run: |
-          python -m pip install --upgrade pip
-          pip install dist/*.whl
-
-      - name: Run tests
-        run: |
-          python -c "import gridData; print(f'Package {gridData.__version__} imported successfully')"
-
   test-pytest:
     name: Run tests
     runs-on: ubuntu-latest
@@ -83,7 +58,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.x"
+          python-version: "3.14"
 
       - name: Download dist files
         uses: actions/download-artifact@v4
@@ -97,14 +72,18 @@ jobs:
           WHEEL_FILE=$(ls dist/*.whl)
           pip install "$WHEEL_FILE"[test]
 
-      - name: Run tests
+      - name: Test import
+        run: |
+          python -c "import gridData; print(f'Package {gridData.__version__} imported successfully')"
+          
+      - name: Run basic tests
         run: |
           pytest --verbose --pyargs gridData
 
   deploy-testpypi:
     name: Deploy to TestPyPI
     runs-on: ubuntu-latest
-    needs: [build, test-install, test-pytest]
+    needs: [build, test-pytest]
     if: |
       github.repository == 'MDAnalysis/GridDataFormats' &&
       (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/'))
@@ -121,14 +100,14 @@ jobs:
           path: dist/
 
       - name: Publish to TestPyPI
-        uses: pypa/gh-action-pypi-publish@v1.12.4
+        uses: pypa/gh-action-pypi-publish@v1.13.0
         with:
           repository-url: https://test.pypi.org/legacy/
 
   deploy-pypi:
     name: Deploy to PyPI
     runs-on: ubuntu-latest
-    needs: [build, test-install, test-pytest]
+    needs: [build, test-pytest]
     if: |
       github.repository == 'MDAnalysis/GridDataFormats' &&
       (github.event_name == 'release' && github.event.action == 'published')
@@ -145,11 +124,15 @@ jobs:
           path: dist/
 
       - name: Publish to PyPI
-        uses: pypa/gh-action-pypi-publish@v1.12.4
+        uses: pypa/gh-action-pypi-publish@v1.13.0
 
   test-deployed-testpypi:
     name: Test deployed package (TestPyPI)
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
     needs: deploy-testpypi
     if: |
       github.repository == 'MDAnalysis/GridDataFormats' &&
@@ -158,7 +141,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.x"
+          python-version: "3.14"
 
       - name: Install from TestPyPI
         run: |
@@ -175,7 +158,11 @@ jobs:
 
   test-deployed-pypi:
     name: Test deployed package (PyPI)
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
     needs: deploy-pypi
     if: |
       github.repository == 'MDAnalysis/GridDataFormats' &&
@@ -184,7 +171,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.x"
+          python-version: "3.14"
 
       - name: Install from PyPI
         run: |

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -13,7 +13,7 @@ The rules for this file:
   * accompany each entry with github issue/PR number (Issue #xyz)
 
 -------------------------------------------------------------------------------
-01/16/2026 IAlibay, ollyfutur, conradolandia, orbeckst, PlethoraChutney, 
+01/22/2026 IAlibay, ollyfutur, conradolandia, orbeckst, PlethoraChutney,
            Pradyumn-cloud
 
   * 1.1.0
@@ -24,10 +24,11 @@ The rules for this file:
     (issue #143)
   * Python 3.13 and 3.14 are now supported (PR #140)
   * Support for Python 3.9 and 3.10 is now dropped as per SPEC0 (PR #140)
+  * all tests and data files are now included in the distribution (#156)
 
   Enhancements
 
-  * `Grid` now accepts binary operations with any operand that can be 
+  * `Grid` now accepts binary operations with any operand that can be
     broadcasted to the grid's shape according to `numpy` broadcasting rules
     (PR #142)
   * `Grid` now allows forcing MRC/CCP4 maps to be read as volumes even when
@@ -38,6 +39,7 @@ The rules for this file:
 
   * Attempting binary operations on grids with different edges now raises an
     exception (PR #142)
+  * updated metadata license identifier (#155)
 
 
 10/21/2023 IAlibay, orbeckst, lilyminium

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -71,6 +71,7 @@ gridData = [
     "tests/datafiles/*.dx",
     "tests/datafiles/*.dx.gz",
     "tests/datafiles/*.ccp4",
+    "tests/datafiles/*.mrc",
     "tests/datafiles/*.plt",
     "tests/datafiles/*.bz2",
 ]

